inst/Description.qml

```diff
@@ -7,6 +7,7 @@ Description
 	title : 		qsTr("Survival (beta)")
 	description:	qsTr("Perform analyses of censored time to event data.")
 	requiresData:	true
+	preloadData:  true
 	icon:			"survival-analysis.svg"
 	version			: "0.19.0"
```

Comment:
up this?

Reply:
what? Do you mean the alignment? It looks aligned for me when I open the file locally